src/components/display/fewlines/StandardIdentitiesSection/StandardIdentitiesSection.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import styled from "styled-components";
+
+import { Button, ButtonVariant } from "../Button/Button";
+import { RightChevron } from "../Icons/RightChevron/RightChevron";
+import {
+  BoxedLink,
+  NoIdentitiesBox,
+} from "../IdentitiesSection/IdentitiesSection";
+import { NeutralLink } from "../NeutralLink";
+import { Separator } from "../Separator/Separator";
+import { ShadowBox } from "../ShadowBox/ShadowBox";
+import { ShowMoreButton } from "../ShowMoreButton/ShowMoreButton";
+import { Identity } from "@lib/@types";
+import { IdentitiesSectionContent } from "@src/@types/identitiesSectionContent";
+
+type StandardIdentitiesSectionProps = {
+  sectionName: string;
+  content: IdentitiesSectionContent;
+  identitiesList: Identity[];
+};
+
+export const StandardIdentitiesSection: React.FC<StandardIdentitiesSectionProps> = ({
+  sectionName,
+  content,
+  identitiesList,
+}) => {
+  const [
+    hideSecondaryIdentities,
+    setHideSecondaryIdentities,
+  ] = React.useState<boolean>(true);
+
+  const { title, noIdentityMessage, addNewIdentityMessage } = content;
+
+  let displayedList: Identity[] = identitiesList;
```

Comment:
Why are you using `let`?

Reply:
I can't reassign it if I use `const`, I do need to reassign this variable thanks to the following `switch` statement